src/components/screens/monitor/blockDetails/index.js

```diff
@@ -0,0 +1,28 @@
+/* istanbul ignore file */
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+import { withRouter } from 'react-router-dom';
+import BlockDetails from './blockDetails';
+import liskService from '../../../../utils/api/lsk/liskService';
+import withData from '../../../../utils/withData';
+
+const mapStateToProps = (state, ownProps) => ({
+  id: ownProps.match.params.id,
+});
+
+export default compose(
+  withRouter,
+  connect(mapStateToProps),
+  withData({
+    blockDetails: {
+      apiUtil: liskService.getBlockDetails,
+      defaultData: [],
```

Comment:
```suggestion
```
IMO `defaultData` for block should be `{}` (just update liskService.getBlockDetails), 
`{}` is the default in https://github.com/LiskHQ/lisk-hub/blob/dd73b2379df1e96828fdadb2097948b1bcfdcd19/src/utils/withData.js#L14 so it's not needed here.